_docs-2/administration/fate.md

```diff
@@ -78,6 +78,10 @@ The command is launched using:
 ```
 > accumulo admin fate --[option]
 ```
+help that lists the fate options (and other admin commands) is available using:
```

Comment:
I feel like we need at least capitalization of the first word here. This could also be reworded as:
> The Accumulo admin help command (`accumulo admin -h`) includes expected usage information.

Reply:
reworded in 945274a0

```
The Accumulo admin help command option `accumulo admin -h` shows the expected usage information for the fate and
other admin commands.
```